data-transformers/transformers/propertiesTransformer.py

```diff
@@ -0,0 +1,269 @@
+            d["oom_kill_disable"] = data[1]
+            continue
+        elif data[0] == "OperatingSystem":
+            d["operating_system"] = data[1]
+            continue
+        elif data[0] == "SwapLimit":
+            d["swap_limit"] = data[1]
+            continue
+        elif data[0] == "SystemTime":
+            d["system_time"] = data[1]
+            continue
+        elif data[0] == "ServerVersion":
+            d["server_version"] = data[1]
+            continue
+        else:
+            continue
+    return d
+
+def createVersionDict(a):
+    ob = json.loads(a.decode())
+    d = {}
+    for data in ob:
+        data = data.split("=")
+        print data
+        if data[0] == "Version":
+            d["docker_version"] = data[1]
+            continue
+        if data[0] == "Os":
+            d["docker_os"] = data[1]
+            continue
+        if data[0] == "ApiVersion":
+            d["docker_api_version"] = data[1]
+            continue
+        if data[0] == "GitCommit":
+            d["docker_git_commit"] = data[1]
+            continue
+        if data[0] == "GoVersion":
+            d["docker_go_version"] = data[1]
+            continue
+        if data[0] == "Arch":
+            d["docker_arch"] = data[1]
+            continue
+        if data[0] == "KernelVersion":
+            d["docker_kernel_version"] = data[1]
+            continue
+    return d
+
+def getFromMinio(url):
+    from commons import getFromUrl
+    return getFromUrl(url)
+
+def hostNeedsSaving(infoData, cassandraKeyspace):
+    hostID = ""
+    
+    ob = json.loads(infoData.decode())
+    for e in ob:
+        e = e.split("=")
+        if e[0] == "ID":
+            hostID = e[1]
+            break
+    
+    if hostID == "":
+        return false
+    
+    hostNotSaved = sc.cassandraTable(cassandraKeyspace, "host_properties") \
+        .select("host_id") \
+        .where("host_id=?", hostID) \
+        .isEmpty()
+        
+    return hostNotSaved
+
+def main():
+    from pyspark_cassandra import CassandraSparkContext
+    from pyspark import SparkConf
+    from pyspark import SparkFiles
+    
+    # Takes arguments
+    minioHost = sys.argv[1]
+    filePath = sys.argv[2]
+    trialID = sys.argv[3]
+    experimentID = sys.argv[4]
+    SUTName = sys.argv[5]
+    containerID = sys.argv[6]
+    table = "container_properties"
+    
+    # Set configuration for spark context
+    conf = SparkConf().setAppName("Properties Transformer")
+    sc = CassandraSparkContext(conf=conf)
+    
+    # Retrieve the configuration file that was sent to spark
+    confPath = SparkFiles.get("data-transformers.yml")
+    with open(confPath) as f:
+        transformerConfiguration = yaml.load(f)
+        cassandraKeyspace = transformerConfiguration["cassandra_keyspace"]
+        minioPort = transformerConfiguration["minio_port"]
+    
+    inspectData = getFromMinio("http://"+minioHost+":"+minioPort+"/"+filePath+"_inspect.gz")[0]  
```

Comment:
Answer to the following, please: https://github.com/benchflow/data-transformers/pull/50/files/4f9492676ef254ee77e9ad986fea2117cd1125e3..363462c7879c685413dead534ec2e70bbc1644f4#r63614569